armi/reactor/assemblies.py

```diff
@@ -206,13 +206,28 @@ def insert(self, index, obj):
 
     def moveTo(self, locator):
         """Move an assembly somewhere else."""
+        oldSymmetryFactor = self.getSymmetryFactor()
         composites.Composite.moveTo(self, locator)
         if self.lastLocationLabel != self.DATABASE:
             self.p.numMoves += 1
             self.p.daysSinceLastMove = 0.0
         self.parent.childrenByLocator[locator] = self
         # symmetry may have changed (either moving on or off of symmetry line)
         self.clearCache()
+        self.scaleParamsToNewSymmetryFactor(oldSymmetryFactor)
+
+    def scaleParamsToNewSymmetryFactor(self, oldSymmetryFactor=1.0):
+        volIntegratedParamsToScale = self.getBlocks()[0].p.paramDefs.atLocation(
+            ParamLocation.VOLUME_INTEGRATED
+        )
+        scalingFactor = oldSymmetryFactor / self.getSymmetryFactor()
+        for b in self.getBlocks():
```

Comment:
Suggestion: I think you can wrap this in an `if scalingFactor != 1` clause, and avoid needlessly looping over a ton of assemblies that were shuffled but not to/from the central assembly.